### PR TITLE
Update location of retrieving domain

### PR DESCRIPTION
**Pull request by JL-LP**
Other API's have the retrieve the domain in the Overview section. Monitoring API has it buried in another page. This change will make it more obvious when people are searching.

### CODE REVIEW

**Comment by j9t** [comment]
@JL-LP, if this is ready to be reviewed, mention me, Itay, or Liat and the respective team member will work with you on getting this out.

**Comment by JL-LP** [comment]
@j9t these changes are ready to be reviewed

File: pages/Documents/WebsiteMonitoring/MonitoringAPI/getting-started.md [hunk 1]
```diff
@@ -9,8 +9,13 @@ order: 10
 permalink: monitoring-api-getting-started.html
 indicator: messaging
 ---
+### Step 1 - Retrieve your domain
 
-### Step 1 — Configure your Conversational Cloud account
+Use the [LivePerson Domain API](agent-domain-domain-api.html) to retrieve this information by providing the following service name:
```
Comment (j9t) [inline pages/Documents/WebsiteMonitoring/MonitoringAPI/getting-started.md:14]: A list for one item seems unusual. How about incorporating this in the text, as with something like:

‘…by providing the service name “smt”.’

or

‘…by providing the following service name: “smt”.’

**Review by j9t — CHANGES_REQUESTED** [review]

File: pages/Documents/WebsiteMonitoring/MonitoringAPI/getting-started.md (hunk 1 above)
Comment (JL-LP) [reply pages/Documents/WebsiteMonitoring/MonitoringAPI/getting-started.md:14]: I was trying to align it with the format that appears everywhere else in our documentation on the developer site such as here for the App Engagement API: https://developers.liveperson.com/app-engagement-api-overview.html

Comment (j9t) [reply pages/Documents/WebsiteMonitoring/MonitoringAPI/getting-started.md:14]: We should change that everywhere, but it’s nothing we’d ask you to do. Fair enough.

**Review by j9t — APPROVED** [review]

**Comment by j9t** [comment]
@JL-LP, good to merge?

**Comment by JL-LP** [comment]
@j9t everything should be good to merge yes

**Pull request merged**